fedora/Dockerfile.amd64

```diff
@@ -1,4 +1,4 @@
-FROM fedora:36
+FROM fedora:38
```

Comment:
Could you also update in `.drone.star`. This will create a separate tag for `38` base and `36` tag will still be available.
https://github.com/owncloud-ci/client/blob/c6b719fedf1e83e4d8d162115762be3429d1da9a/.drone.star#L17-L20